### PR TITLE
Importing certificate to power device IPDU if auto_import_cert is true

### PR DESCRIPTION
**Pull request by ricardogpsf**
### Description
Importing certificate to power device IPDU if auto_import_cert property is set to true.
When we are applying discover action to oneview_power_device, some appliances (like appliance of OV 3.10) require the valid certificate to access IPDU server, this commit changes are inserting the code to import the certificate;

### Environment Details
 - **oneview Cookbook:** Version: 2.3.0
 - **OneView Version:** 3.10
 - **chef-client Version:** 12.0+
 - **platform:** Any

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
  - [ ] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [ ] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -687,6 +687,7 @@ oneview_power_device '<iPDU hostname>' do
   client <my_client>
   username <username>
   password <password>
+  auto_import_cert [true, false] # Optional. Only used with the :discover action. Default value is false.
```
Comment (fgbulsoni) [inline README.md]: I guess that if we're really trying to make things easier to the end user here, `true` would be the better default.

Comment (aalexmonteiro) [inline README.md]: Suggestion: Maybe it was good that the name was ```import_certificate```. It is more readable.

**Review by fgbulsoni — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fgbulsoni) [reply README.md]: I do like `import_certificate`, but I feel it is not clear that it is doing so automatically. 

In the case of `import_certificate` I'd expect a field to pass in the certificate.
Perhaps `auto_import_certificate` ? 

My naming sense sucks though, If you guys don't feel like it lacks clarity `import_certificate` is also fine by me.

Comment (ricardogpsf) [reply README.md]: I was thinking in client that are using "old" appliances, that don't need import some certificate, they will have to do changes into theirs recipes of power_device, right?! Is this the better way?

Comment (ricardogpsf) [reply README.md]: I'll change the name to `auto_import_certificate`, right?! =D \o/

File: libraries/resource_providers/api200/power_device_provider.rb [hunk 2]
```diff
@@ -23,6 +23,8 @@ def discover
             pd_klass.discover(@item.client, hostname: @name, username: @new_resource.username, password: @new_resource.password)
           rescue OneviewSDK::OneViewError => error
             raise error unless @new_resource.auto_import_certificate && error.message.include?('Unable to retrieve the input certificate')
+            Chef::Log.warn("Oneview returned the following error:\n: #{error.message}")
+            Chef::Log.info("Trying to import certificate for #{@resource_name} '#{@name}'")
```
Comment (fgbulsoni) [inline libraries/resource_providers/api200/power_device_provider.rb]: Trying to **automatically** import (...)

**Review by fgbulsoni — APPROVED** [review]

**Pull request merged**